### PR TITLE
Fixed some but missing | % in few places

### LINKED ISSUE #1740: Pester crashes if user does not have alias `foreach` defined
## 1. General summary of the issue

Invoke Pester crashes if the user's profile contains this:
```powershell
Remove-Alias -Name 'foreach' -Force
```

## 2. Describe Your Environment

```powershell
Pester version     : 5.0.4 C:\Users\cppmo_000\Documents\PowerShell\Modules\Pester\5.0.4\Pester.psd1
PowerShell version : 7.0.3
OS version         : Microsoft Windows NT 10.0.19041.0
```

## 3. Expected Behavior
`Invoke-Pester` should run without the user requiring user aliases.

## 4.Current Behavior

```powershell
Invoke-Pester

System.Management.Automation.CommandNotFoundException: The term 'foreach' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
   at System.Management.Automation.ExceptionHandlingOps.CheckActionPreference(FunctionContext funcContext, Exception exception)    
   at System.Management.Automation.Interpreter.ActionCallInstruction`2.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
at Invoke-Pester<End>, C:\Users\user\Documents\PowerShell\Modules\Pester\5.0.4\Pester.psm1: line 4501
at <ScriptBlock>, <No file>: line 
```

## 5. Possible Solution

Running `PSScriptAnalyzer` with `PSAvoidUsingCmdletAliases` to find the `ForEach-Object`s

### PR DESCRIPTION
**Pull request by nohwnd**
Fix #1740

### CODE REVIEW

File: src/Format.psm1 [hunk 1]
```diff
@@ -10,7 +10,7 @@ function Format-Collection ($Value, [switch]$Pretty) {
     }
     $count = $Value.Count
     $trimmed = $count  -gt $Limit
-    '@(' + (($Value | Select -First $Limit | % { Format-Nicely -Value $_ -Pretty:$Pretty }) -join $separator) + $(if ($trimmed) {', ...'}) + ')'
```
Comment (fflaten) [inline src/Format.psm1:13]: Can we guarantee that `$Value` is not an empty collection? `IndexOutOfRangeException` on `@()[0..0]` in strictmode. Same in `Sort-Property`

**Comment by fflaten** [comment]
Not to step on any toes, but I actually fixed all aliases and unsafe commands in `/src/*` in https://github.com/fflaten/Pester/tree/analyzercleanup while testing the custom rule. Solved `Format.psm1` by importing the safecommands-script in the if not build-clause, so all tests pass locally now if you'd like to save some time.

Doesn't replace the other edits here though. 👍

**Comment by nohwnd** [comment]
No toes were stepped on. Please PR it.

File: src/Format.psm1 (hunk 1 above)
Comment (nohwnd) [reply src/Format.psm1:13]: not this way. I did not think about that. 👍

**Comment by nohwnd** [comment]
abandoning.

**Pull request closed without merging**